lib/toggle.js

```diff
@@ -62,23 +63,32 @@ const ToggleBody = styled.label`
   width: calc(var(--space-4) + var(--space-2));
   vertical-align: middle;
   overflow: hidden;
+  opacity: ${({ disabled }) => (disabled ? 0.5 : 1)};
   &:focus-within {
-    box-shadow 0 0 4px blue;
+    box-shadow: 0 0 4px blue;
   }
 `;
 
-export const Toggle = ({ value, onChange, label, ...props }) => (
-  <ToggleBody {...props}>
+export const Toggle = ({ value, onChange, label, disabled, trueLabel, falseLabel, ...props }) => (
+  <ToggleBody {...props} disabled={disabled}>
     <VisuallyHidden>
-      <input type="checkbox" checked={value} onChange={(e) => onChange(e.target.checked)} aria-label={label} />
+      <input type="checkbox" checked={value} onChange={(e) => onChange(e.target.checked)} aria-label={label} disabled={disabled} />
     </VisuallyHidden>
-    <Shuttle active={value} aria-hidden="true" />
+    <Shuttle active={value} aria-hidden="true" trueLabel={trueLabel} falseLabel={falseLabel} />
   </ToggleBody>
 );
 Toggle.propTypes = {
   value: PropTypes.any.isRequired,
   onChange: PropTypes.func.isRequired,
   label: PropTypes.string.isRequired,
```

Comment:
all these label names are a bit confusing now. I think this label is only used by `aria-label` now, right - shall we just call the parameter by the same name to make it clear (or `input-aria-label`)? 

It is also a bit odd for the example that's provided in the story, because it separately has a "Notifications Settings" label, so I think we should really be passing in `aria-labelledby` for that example, with the ID of that label (per https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute). We could leave that for a future PR though; perhaps a general accessibility sweep is needed.